### ISSUE TITLE
förslag: saknar komplett lista med symboler av blockschema?

### ISSUE THREAD

**Issue by cicci44** [SverigesSandareamatorer/SSA-Akademin#638]
inom programmeringen använder vi fbd (function block diagram) och de följer en viss standard. Bygger man funktioner som blir för avancerade, skapar man ett nytt funktionsblock med det avancerade innehållet, döper om det, lägger in lämplig symbol, kort beskrivning och laddar upp den till ett bibliotek. 

....vore toppen att komplettera någon form av lista med de vanligaste symboler man använder i blockschema och ifall det finns en standard att hänvisa till.

**Comment by phieri** [comment]
Välkommen och tack för återkopplingen! Låter rimligt med en förklaring av hur blocken väljs och designas. Texten i 6.1.1 Blockschema lämnar den biten öppen som det är nu.

**Comment by phieri** [comment]
T/R 61-02 listar följande:
>4.3 Operation and function of the following stages (Block diagram treatment only)
>- HF amplifier [with tuned or fixed band pass];
>- Oscillator [fixed and variable];
>- Mixer;
>- Intermediate frequency amplifier;
>- Limiter;
>- Detector, including product detector;
>- Audio amplifier;
>- Automatic gain control;
>- S meter;
>- Squelch.

samt

>5.3 Operation and functions of the following stages (Block diagram treatment only)
>- Mixer;
>- Oscillator;
>- Buffer;
>- Driver;
>- Frequency multiplier;
>-- Power amplifier;
>- Output matching;
>- Output filter;
>- Frequency modulator;
>- SSB modulator;
>- Phase modulator;
>- Crystal filter.

Vi kanske borde se till att dessa symboler finns presenterade.

**Comment by cicci44** [comment]
som notering till tidigare förslag, så har SSAs övningsprov annorlunda blockschema än de som finns redovisade i konCEPT. dvs
![image](https://github.com/SverigesSandareamatorer/SSA-Akademin/assets/1296978/0f41cd12-0739-4d32-8b56-9fe5cf0ee95f)
och
![image](https://github.com/SverigesSandareamatorer/SSA-Akademin/assets/1296978/f7d29eb4-b997-43ac-bf54-a0901721bf1c)

andra exempel är..
![image](https://github.com/SverigesSandareamatorer/SSA-Akademin/assets/1296978/ad818851-0f4a-44f4-96a4-aeeb2ca335f8)
och
![image](https://github.com/SverigesSandareamatorer/SSA-Akademin/assets/1296978/5bbafadc-3750-4ff5-8c9b-a84cb704aa6e)

**Comment by sikvall** [comment]
Ja, vi bör ha samma ikonografi och symbolik, sen om de på proven a ses
arkaiska kan vi ju ha de andra varianterna bredvid men huvudsakligen
använda samma som provet.

Alternativet är att uppdatera proven om det är de som har de mer arkaiska
symbolerna, det kan jag här och nu inte avgöra.

Det blandas ju också lite mellan europeiska och amerikanska symboler vilket
egentligen är ok, man kommer stöta på båda sorterna, men ett större tag om
detta vore också fint.

Anders

Den ons 10 maj 2023 11:43cicci44 ***@***.***> skrev:

**Comment by sm5phu** [comment]
Jonas bollar förslag på standard mot "SSA centralt".
